app/authorities/qa/authorities/format.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Qa::Authorities
+  # Format QA Object
+  class Format < Qa::Authorities::Base
+    include WebServiceBase
+
+    class_attribute :label
+
+    self.label = lambda do |item|
+      [item.first['http://www.w3.org/2000/01/rdf-schema#label'].first['@value']].compact.join(', ')
+    end
+
+    def search(q)
+      parse_authority_response(json(build_query_url(q)))
+    end
+
+    def build_query_url(q)
+      CGI.escape(q)
```

Comment:
CGI breaks for me, URI is fine

Reply:
Better yet, allowing users to arbitrarily force us to hit any uri doesn't seem smart.
This should become:
```
query = CGI.escape(q)
"https://w3id.org/spar/mediatype/#{q}"
```
Then users only input `image/tiff` at the form and their autocomplete comes up, and we're guaranteed to hit the right vocab.

Reply:
CGI is still overbearing with escaping slashes. Also, QA doesn't automatically retrieve redirects. I had to solve the latter for some work with another vocab, so I'll push up my fix for that and will include something for slashes.